testing-internal/src/main/java/com/linecorp/armeria/internal/testing/ImmediateEventLoop.java

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright 2023 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.internal.testing;
+
+import io.netty.channel.DefaultEventLoop;
+import io.netty.channel.EventLoop;
+
+public final class ImmediateEventLoop extends DefaultEventLoop {
```

Comment:
Can we also override `invokeAny` or `invokeAll` because it throws an exception?
I think we can also just add a limitation with a comment because it's an internal class.

Reply:
I think it makes more sense to implement those methods when a test cases actually needs them since implementing them to execute immediately requires certain assumptions that may differ between requirements.
Added a comment instead 🙇